proto/iotics/api/host.proto

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) 2019-2022 Iotic Labs Ltd. All rights reserved.
+
+// Iotics Web protocol definitions (hosts)
+syntax = "proto3";
+
+import "iotics/api/common.proto";
+
+package iotics.api;
+
+option csharp_namespace = "Iotics.Api";
+option go_package = "github.com/Iotic-Labs/iotic-go-proto-qapi/iotics/api;ioticsapi";
+option java_multiple_files = true;
+option java_outer_classname = "HostProto";
+option java_package = "com.iotics.api";
+option objc_class_prefix = "IAX";
+option php_namespace = "Iotics\\Api";
+
+// ---------------------------------------------------------------------------------------------------------------------
+
+
+// HostAPI enables management of Iotics host twins.
+// Services only affect local resources, unless stated otherwise.
+service HostAPI {
+  // GetHostTwinID gets the ID of the host twin.
+  rpc GetHostTwinID(GetHostTwinIDRequest) returns (GetHostTwinIDResponse) {}
```

Comment:
It would be good to maintain the consistency and call this `HostTwinDid`.

Reply:
@mihai-dinculescu at the moment it will return PeerID though, until @conbon makes some other changes

Reply:
So what I mean to say is `HostTwinID` vs `HostTwinDID`?

Reply:
I thought we were going with `hostId` so the switchover is transparent and it matches with the coming api changes

Reply:
If it's still peer id, then yes, `HostTwinID` makes sense.